zeus/optimizer/batch_size/server/models.py

```diff
@@ -0,0 +1,95 @@
+"""Shared model definitions for the server and client."""
+
+from enum import Enum
+from typing import Any, List, Dict
+from uuid import UUID
+
+from pydantic import (
+    BaseModel,
+    root_validator,
+    validator,
+)
+
+
+class MabSetting(BaseModel):
+    """Mab setting"""
```

Comment:
Later please make the formatting of docstrings consistent (e.g., noun phrase with reasonable detail, ends with period), `Args:` section that explains what each fields are, etc.